detectable/src/main/java/com/synopsys/integration/detectable/detectables/pip/PipInspectorDetectable.java

```diff
@@ -69,12 +76,21 @@ public PipInspectorDetectable(final DetectableEnvironment environment, final Fil
     @Override
     public DetectableResult applicable() {
         setupFile = fileFinder.findFile(environment.getDirectory(), SETUPTOOLS_DEFAULT_FILE_NAME);
-        final boolean hasSetups = setupFile != null;
-        final boolean hasRequirements = pipInspectorDetectableOptions.getRequirementsFilePaths() != null && pipInspectorDetectableOptions.getRequirementsFilePaths().size() > 0;
+        boolean hasSetups = setupFile != null;
+
+        requirementsFiles = pipInspectorDetectableOptions.getRequirementsFilePaths();
+        if (CollectionUtils.isEmpty(pipInspectorDetectableOptions.getRequirementsFilePaths())) {
+            requirementsFiles = fileFinder.findFiles(environment.getDirectory(), REQUIREMENTS_DEFAULT_FILE_NAME)
```

Comment:
Does the current file finder take into account search depth? I know some customers are going to have requirements files not at root level, for that we could just say use the property.

Reply:
Hmm it doesn't. That seems odd. I think it's reasonable to expect that we'd respect search paths for this, I'll add that.

Reply:
I talked with Jordan about it. We think what you already have is good

Reply:
Yeah, I think ideally we should respect search depth here but I'm not keen on adding another property.